src/main/java/com/iexec/sms/admin/AdminController.java

```diff
@@ -32,6 +32,20 @@
 @RestController("/admin")
 public class AdminController {
 
+    /**
+     * The directory where the database backup file will be stored.
+     * The value of this constant should be a valid, existing directory path.
+     */
+    private static final String BACKUP_STORAGE_LOCATION = "/work/";
+
+    /**
+     * The name of the database backup file.
+     */
+    private static final String BACKUP_FILENAME = "backup.sql";
+
+    /**
+     * We want to perform one operation at a time. This  ReentrantLock is used to set up the lock mechanism.
```

Comment:
One unnecessary space there :)
```suggestion
     * We want to perform one operation at a time. This ReentrantLock is used to set up the lock mechanism.
```

Reply:
done in local